extensions/wikia/Recirculation/js/experiments/mix.js

```diff
@@ -0,0 +1,135 @@
+/*global require*/
+require([
+	'jquery',
+	'wikia.window',
+	'wikia.abTest',
+	'wikia.log',
+	'ext.wikia.recirculation.tracker',
+	'ext.wikia.recirculation.utils',
+	'ext.wikia.recirculation.discussions',
+	'ext.wikia.recirculation.tracker',
+	require.optional('videosmodule.controllers.rail')
+], function(
+	$,
+	w,
+	abTest,
+	log,
+	tracker,
+	utils,
+	discussions,
+	tracker,
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename the duplicated function parameter "tracker". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ADuplicateFunctionArgument)